plugins/scaffolder-backend/src/scaffolder/actions/builtin/fetch/index.ts

```diff
@@ -16,3 +16,4 @@
 
 export { createFetchPlainAction } from './plain';
 export { createFetchCookiecutterAction } from './cookiecutter';
+export * from './helpers';
```

Comment:
should use named exports, not star, in the "leaf" index files https://backstage.io/docs/architecture-decisions/adrs-adr004